src/Microsoft.Bcl.HashCode/tests/Microsoft.Bcl.HashCode.Tests.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFramework>net461</TargetFramework>
+      <IncludeRemoteExecutor>true</IncludeRemoteExecutor>

```

Comment:
Remove this property as you are bringing in RemoteExecutor via a PackageReference below.